In [1]:
import pandas as pd
#pymysql.install_as_MySQLdb()

In [2]:
a = pd.read_csv("gene_condition_source_id", sep = "\t")
a

#GeneID AssociatedGenes RelatedGenes ConceptID  \
0            2             A2M          NaN  C0002395   
1            2             A2M          NaN  C3279661   
2       144568           A2ML1          NaN  C1833692   
3        53947          A4GALT          NaN  C3549485   
4         8086            AAAS          NaN  C0271742   
5        79719           AAGAB          NaN  C1835662   
6           16           AARS1          NaN  C2750090   
7           16           AARS1          NaN  C4225361   
8           16             NaN        AARS1  C0007959   
9           16             NaN        AARS1  C0007959   
10          16             NaN        AARS1  C0007959   
11          16             NaN        AARS1  C0270914   
12       57505           AARS2          NaN  C3279793   
13       57505           AARS2          NaN  C4014588   
14       57505             NaN        AARS2  CN228601   
15       10157            AASS          NaN  C0268556   
16       10157            AASS          NaN  C0268556   
17       10157            AASS          NaN  C0543533   
18       10157            AASS          NaN  C0543533   
19       10157            AASS          NaN  C0543533   
20       10157            AASS          NaN  C0543533   
21       10157            AASS          NaN  C0543533   
22          18            ABAT          NaN  C0342708   
23          19           ABCA1          NaN  C0039292   
24          19           ABCA1          NaN  C0039292   
25          19           ABCA1          NaN  C1704429   
26       26154          ABCA12          NaN  C0598226   
27       26154          ABCA12          NaN  C1832550   
28       26154             NaN       ABCA12  C0020758   
29          21           ABCA3          NaN  C1970456   
...        ...             ...          ...       ...   
11340    51364             NaN      ZMYND10  C0008780   
11341    10771         ZMYND11          NaN  C4015167   
11342    84225         ZMYND15          NaN  C4014454   
11343     7700          ZNF141          NaN  C3808889   
11344     7701          ZNF142          NaN  CN258385   
11345     7702          ZNF143          NaN  CN043592   
11346     7707          ZNF148          NaN  C4310644   
11347    63925          ZNF335          NaN  C3554499   
11348    63925             NaN       ZNF335  C3711387   
11349    84905          ZNF341          NaN  C4748969   
11350    22891          ZNF365          NaN  C2700426   
11351    79797          ZNF408          NaN  C4225315   
11352    79797          ZNF408          NaN  C4225316   
11353    23090          ZNF423          NaN  C3539071   
11354    58499          ZNF462          NaN  CN262386   
11355    84627          ZNF469          NaN  C0268344   
11356   130557          ZNF513          NaN  C3150879   
11357   130557             NaN       ZNF513  C0035334   
11358   130557             NaN       ZNF513  C0035334   
11359   130557             NaN       ZNF513  C0035334   
11360    84146          ZNF644          NaN  C3279997   
11361    57592          ZNF687          NaN  C4085250   
11362     7552          ZNF711          NaN  C2749020   
11363    79755          ZNF750          NaN  C1853258   
11364     9326          ZNHIT3          NaN  C1850055   
11365    22917             ZP1          NaN  C4014291   
11366     7783             ZP2          NaN  CN258242   
11367     7784             ZP3          NaN  C4540205   
11368    57688          ZSWIM6          NaN  C1863616   
11369    57688          ZSWIM6          NaN  C4693405   

                                             DiseaseName  \
0                                    Alzheimer's disease   
1                       Alpha-2-macroglobulin deficiency   
2                        Otitis media, susceptibility to   
3                                            p phenotype   
4               Glucocorticoid deficiency with achalasia   
5                      Keratosis palmoplantaris papulosa   
6                   Charcot-Marie-Tooth disease, typ

In [3]:
dis = set(a["DiseaseName"][10:65]) 
print(len(dis))
genes =  set(a["AssociatedGenes"][11:230]   )
genes1 = [] 
print(len(genes))
for g in genes:
    if(g==g):
        genes1.append(g)
dis1=[]
for d in dis:
    if(d==d):
        dis1.append(d)

41
81


In [4]:
a =pd.DataFrame.dropna(a[["AssociatedGenes", "DiseaseName"]][10:65])
a

AssociatedGenes                                        DiseaseName
12           AARS2    Combined oxidative phosphorylation deficiency 8
13           AARS2  Leukoencephalopathy, progressive, with ovarian...
15            AASS                                    Saccharopinuria
16            AASS                                    Saccharopinuria
17            AASS                                     Hyperlysinemia
18            AASS                                     Hyperlysinemia
19            AASS                                     Hyperlysinemia
20            AASS                                     Hyperlysinemia
21            AASS                                     Hyperlysinemia
22            ABAT    Gamma-aminobutyric acid transaminase deficiency
23           ABCA1                                    Tangier disease
24           ABCA1                                    Tangier disease
25           ABCA1                  Familial hypoalphalipoproteinemia
26          ABCA12       Autosomal recessive congenital ichthyosis 4B
27          ABCA12       Autosomal recessive congenital ichthyosis 4A
29           ABCA3    Surfactant metabolism dysfunction, pulmonary, 3
31           ABCA4                                Stargardt disease 1
32           ABCA4                               Cone-rod dystrophy 3
33           ABCA4                            Retinitis pigmentosa 19
34           ABCA4                 Age-related macular degeneration 2
41           ABCA5          Gingival fibromatosis with hypertrichosis
42           ABCA7                          Alzheimer disease, type 9
44           ABCB1                              Colchicine resistance
45           ABCB1                      Inflammatory bowel disease 13
46          ABCB11        Benign recurrent intrahepatic cholestasis 2
47          ABCB11    Progressive familial intrahepatic cholestasis 2
49           ABCB4    Progressive familial intrahepatic cholestasis 3
50           ABCB4                                      Cholecystitis
51           ABCB4                                      Cholecystitis
52           ABCB4                                      Cholecystitis
53           ABCB4          Cholestasis, intrahepatic, of pregnancy 3
54           ABCB6  Pseudohyperkalemia, familial, 2, due to red ce...
55           ABCB6                              Langereis blood group
56           ABCB6          Microphthalmia, isolated, with coloboma 7
57           ABCB6           Dyschromatosis universalis hereditaria 3
58           ABCB7    Anemia sideroblastic and spinocerebellar ataxia
59          ABCC11             Apocrine gland secretion, variation in
60           ABCC2                             Dubin-Johnson syndrome
61           ABCC2                             Dubin-Johnson syndrome
62           ABCC2                             Dubin-Johnson syndrome
63           ABCC6                           Pseudoxanthoma elasticum
64           ABCC6                           Pseudoxanthoma elasticum

In [5]:
b = 

SyntaxError: invalid syntax (<ipython-input-5-51da195f42cb>, line 1)

In [6]:
insert into Variant(idVariant, Chromosome, Posiion) values( "A2M", "2", "12190");

SyntaxError: invalid syntax (<ipython-input-6-c331440ab8b0>, line 1)

In [7]:
import random 
chroms = [ str(i) for i in range(1,23)]
chroms.append("X")
chroms.append("Y")
s = random.choice(chroms)
s

'18'

In [8]:
random.uniform(100100, 900009)

338051.0298947984

KeyError: 'nan'

In [107]:
q =a["AssociatedGenes"]
q.iloc([1])

TypeError: unhashable type: 'list'

id: с 11 +16, 84+10, 124+20

In [74]:
all_s = ""
ss_to_db = []
start_ind =  124
nucls = ["A","T","G","C"]
for i in range(20):
    
    num_vars = int(random.uniform(2,20))
    used_vars = []
    
    for attempt in range(num_vars):
        to_gt = ""
        d = random.choice(genes1)
        if d in used_vars:
            continue
        dp = int(random.uniform(3,150))
        num_vars1 = int(random.uniform(0,10))
        num_vars2 = int(random.uniform(1,10))
        if num_vars1==0:
            to_gt=to_gt+"-/"
        else:
            for i in range(num_vars1):
                to_gt=to_gt+random.choice(nucls)
            to_gt=to_gt+"/"
        for i in range(num_vars2):
            to_gt=to_gt+random.choice(nucls)
        s1 = 'insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values('+ str(start_ind)+','+str(dp)+ ',"' +d+'","'+to_gt + '");'   
    
        ss_to_db.append(s1)
        used_vars.append(d)
    start_ind = start_ind+1        
        
    
    #d = random.choice(genes1)
    #chrom = random.choice(["Exome", "Panel"])
    #freq =random.choice(pols)
    
    #insert into Phenotype (Disease_name, Invalid_status) values( "Alzheimer's disease", "I");
    #s1 = 'insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values('+ str(start_ind)+','+str(dp) ',"' +d+'","'+freq + '");'   
    
    #ss_to_db.append(s1)
    #start_ind = start_ind+1
    
    

In [75]:
all_s = ""
for s in ss_to_db:
    all_s=all_s+s

In [76]:
all_s

'insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,5,"ACP2","TGGGA/TTCTCAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,31,"ABHD12","AG/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,145,"ABCA7","CAGGT/GT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,79,"ACSL4","TTG/GAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,70,"ACP5","TA/AAAACCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,78,"ACTB","T/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,25,"ACTN2","T/TGATACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,42,"ADAMTS17","CGT/GTCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,114,"ACTL6B","ACTC/TCCAGGGTG");insert into Sample_variant(Sample_id,DP, Variant_idVa

In [32]:
import random 
all_s = ""
ss_to_db = []
for i in range(256):
    


    #chrom=random.choice(chroms)
    #position = str(int(random.uniform(100100, 900009)))
    #print(position)
    #print(chrom)
    d = random.choice(genes1)
    chrom = random.choice(pols[0:2])
    freq = str(random.uniform(0.00001, 0.15))
    
    #insert into Phenotype (Disease_name, Invalid_status) values( "Alzheimer's disease", "I");
    s1 = 'insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("'+ d+ '","' +chrom +'",'+freq + ');'   
    ss_to_db.append(s1)
    

In [33]:
len(ss_to_db )

256

In [34]:
all_s = ""
for s in ss_to_db:
    all_s=all_s+s

In [35]:
all_s

'insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABHD12","Russian",0.07257190582913159);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCB4","Tatar",0.12610843388606388);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA3","Russian",0.1457100417707173);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACHE","Tatar",0.04854405373977224);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTL6B","Tatar",0.026909062477879386);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABL1","Tatar",0.09831949236309123);insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTL6B","Tatar",0.028233374472624726);insert into Population_variant(Variant_idVariant, Popul

insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("AASS","Russian","0.1352481432236671");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTN3","Chechen","0.033543846709193664");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS13","Chechen","0.07555711424039585");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCC6","Chuvash","0.003980263029715156");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABHD5","Chuvash","0.008195648908449513");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTA1","Other","0.08191145283693911");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA5","Other","0.005142563805581547");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADA","Chechen","0.00965041396667429");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACADM","Armenian","0.03363168257579765");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS18","Chuvash","0.04677970874165376");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTA1","Russian","0.1214954304175831");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACOX1","Other","0.1290593122209436");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTB","Ukrainian","0.09587672102854936");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCB6","Tatar","0.0206809909764784");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACAD9","Ukrainian","0.12955479798231337");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA3","Ukrainian","0.033313778427954396");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACE","Ukrainian","0.09821578847396786");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABHD12","Tatar","0.12171117128757744");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACADM","Chechen","0.03146379128012346");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCB4","Tatar","0.13229376377867283");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS2","Tatar","0.0484872235177742");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS13","Tatar","0.07932513906380037");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCD3","Bashkir","0.13589918044788069");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACOX1","Chechen","0.1168282893277969");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA5","Tatar","0.12443406463745113");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACADS","Chechen","0.08236366032830009");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA7","Bashkir","0.03302023494640281");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTL6B","Other","0.07256762685846582");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTN4","Chuvash","0.13501454110299724");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCC9","Russian","0.0718478470742733");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACACA","Ukrainian","0.14965066148463235");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACVR2B","Tatar","0.09958880116899337");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTN4","Chuvash","0.10910138232363498");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTG1","Tatar","0.053141934788880685");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACSF3","Russian","0.06669155362558483");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACADS","Ukrainian","0.018418762968303158");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACAD8","Chechen","0.09086471018084673");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCC6","Chuvash","0.01850920082213219");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTG2","Chuvash","0.09823466605078174");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCC2","Other","0.02195831966488927");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACVR1","Armenian","0.03274878103936497");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACAD9","Bashkir","0.12246840003399448");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACKR1","Bashkir","0.09408214999684615");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACER3","Armenian","0.061873966551278345");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACADM","Other","0.03779755910729412");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS10","Bashkir","0.0667175780950502");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTC1","Chechen","0.09598538898594522");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACKR1","Tatar","0.13048137633565662");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCB1","Other","0.07837922109258705");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCA12","Ukrainian","0.025029424471396642");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCG8","Other","0.14480362665383478");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACTC1","Armenian","0.011014051537794958");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ACY1","Chechen","0.09109930597665475");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ADAMTS10","Bashkir","0.11899548698329418");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABHD5","Chuvash","0.07318489825881037");insert into Population_variant(Variant_idVariant, Population_Population_name, Frequency ) values("ABCB4","Other","0.03401054138167776");

insert into Phenotype (Disease_name, Invalid_status) values("Alpha-methylacyl-CoA racemase deficiency","I");insert into Phenotype (Disease_name, Invalid_status) values("Leber congenital amaurosis 4","I");insert into Phenotype (Disease_name, Invalid_status) values("Infantile-onset ascending hereditary spastic paralysis","I");insert into Phenotype (Disease_name, Invalid_status) values("Plasma triglyceride level quantitative trait locus","I");insert into Phenotype (Disease_name, Invalid_status) values("Neuroblastoma","I");insert into Phenotype (Disease_name, Invalid_status) values("Joubert syndrome 3","II");insert into Phenotype (Disease_name, Invalid_status) values("Hereditary spherocytosis","I");insert into Phenotype (Disease_name, Invalid_status) values("Pyridoxine-dependent epilepsy","II");insert into Phenotype (Disease_name, Invalid_status) values("SPERMATOGENIC FAILURE 27","I");insert into Phenotype (Disease_name, Invalid_status) values("Congenital disorder of glycosylation type 1P","II");insert into Phenotype (Disease_name, Invalid_status) values("Combined oxidative phosphorylation deficiency","I");insert into Phenotype (Disease_name, Invalid_status) values("Conduction disorder of the heart","II");insert into Phenotype (Disease_name, Invalid_status) values("Cutis laxa-corneal clouding-oligophrenia syndrome","I");insert into Phenotype (Disease_name, Invalid_status) values("Neoplasm of ovary","I");insert into Phenotype (Disease_name, Invalid_status) values("Combined oxidative phosphorylation deficiency 6","I");insert into Phenotype (Disease_name, Invalid_status) values("Myasthenic syndrome, congenital, 14","PO");insert into Phenotype (Disease_name, Invalid_status) values("Enlarged parietal foramina","I");insert into Phenotype (Disease_name, Invalid_status) values("Frontonasal dysplasia 2","II");insert into Phenotype (Disease_name, Invalid_status) values("Joubert syndrome","I");insert into Phenotype (Disease_name, Invalid_status) values("Non-ketotic hyperglycinemia","PO");insert into Phenotype (Disease_name, Invalid_status) values("Adenylate kinase deficiency, hemolytic anemia due to","II");insert into Phenotype (Disease_name, Invalid_status) values("Beta-aminoisobutyric aciduria","I");insert into Phenotype (Disease_name, Invalid_status) values("Reticular dysgenesis","I");insert into Phenotype (Disease_name, Invalid_status) values("Amyotrophic lateral sclerosis","II");insert into Phenotype (Disease_name, Invalid_status) values("Juvenile amyotrophic lateral sclerosis","II");insert into Phenotype (Disease_name, Invalid_status) values("Long QT syndrome 11","I");insert into Phenotype (Disease_name, Invalid_status) values("Pontocerebellar hypoplasia, type 9","I");insert into Phenotype (Disease_name, Invalid_status) values("Hereditary fructosuria","I");insert into Phenotype (Disease_name, Invalid_status) values("Ischemic stroke","I");insert into Phenotype (Disease_name, Invalid_status) values("Epileptic encephalopathy, early infantile, 36","I");insert into Phenotype (Disease_name, Invalid_status) values("Porphobilinogen synthase deficiency","I");insert into Phenotype (Disease_name, Invalid_status) values("Congenital disorder of glycosylation type 1D","I");insert into Phenotype (Disease_name, Invalid_status) values("Osteopathia striata with cranial sclerosis","I");insert into Phenotype (Disease_name, Invalid_status) values("Charcot-Marie-Tooth disease","I");insert into Phenotype (Disease_name, Invalid_status) values("Schizophrenia","I");insert into Phenotype (Disease_name, Invalid_status) values("Glycogen storage disease","II");insert into Phenotype (Disease_name, Invalid_status) values("Cowden syndrome 6","PO");insert into Phenotype (Disease_name, Invalid_status) values("Microphthalmia, isolated 8","PO");insert into Phenotype (Disease_name, Invalid_status) values("Amyotrophic lateral sclerosis type 9","I");insert into Phenotype (Disease_name, Invalid_status) values("Renal dysplasia","II");insert into Phenotype (Disease_name, Invalid_status) values("Parietal foramina 2","I");

'insert into Variant(idVariant, Chromosome, Posiion) values("AMPD2","3","142719");insert into Variant(idVariant, Chromosome, Posiion) values("ALOXE3","8","387157");insert into Variant(idVariant, Chromosome, Posiion) values("AHI1","8","223210");insert into Variant(idVariant, Chromosome, Posiion) values("ALS2","4","715978");insert into Variant(idVariant, Chromosome, Posiion) values("AMT","8","688175");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH3A2","12","554721");insert into Variant(idVariant, Chromosome, Posiion) values("ALOX5","21","196494");insert into Variant(idVariant, Chromosome, Posiion) values("AICDA","8","375110");insert into Variant(idVariant, Chromosome, Posiion) values("ALG10B","13","185966");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH5A1","2","616344");insert into Variant(idVariant, Chromosome, Posiion) values("ALG14","18","487972");insert into Variant(idVariant, Chromosome, Posiion) values("ALG1","4","258068");insert into Variant(idVariant, Chromosome, Posiion) values("AKT1","17","406029");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH18A1","Y","619392");insert into Variant(idVariant, Chromosome, Posiion) values("AMTN","17","147561");insert into Variant(idVariant, Chromosome, Posiion) values("AHCY","9","600142");insert into Variant(idVariant, Chromosome, Posiion) values("AKR1C4","22","484905");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH6A1","14","799317");insert into Variant(idVariant, Chromosome, Posiion) values("AK2","Y","398082");insert into Variant(idVariant, Chromosome, Posiion) values("AGXT","16","166412");insert into Variant(idVariant, Chromosome, Posiion) values("AHDC1","14","570951");insert into Variant(idVariant, Chromosome, Posiion) values("AGRP","12","273378");insert into Variant(idVariant, Chromosome, Posiion) values("AMPD3","21","493686");insert into Variant(idVariant, Chromosome, Posiion) values("AMBN","8","579605");insert into Variant(idVariant, Chromosome, Posiion) values("ALKBH8","X","792796");insert into Variant(idVariant, Chromosome, Posiion) values("ALG8","11","202214");insert into Variant(idVariant, Chromosome, Posiion) values("ALG13","3","830079");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH4A1","3","323465");insert into Variant(idVariant, Chromosome, Posiion) values("AMHR2","9","737632");insert into Variant(idVariant, Chromosome, Posiion) values("ALAS2","7","708107");insert into Variant(idVariant, Chromosome, Posiion) values("AHSG","12","135218");insert into Variant(idVariant, Chromosome, Posiion) values("AK7","18","366391");insert into Variant(idVariant, Chromosome, Posiion) values("AKAP10","16","215259");insert into Variant(idVariant, Chromosome, Posiion) values("ALAD","18","140818");insert into Variant(idVariant, Chromosome, Posiion) values("ALPL","2","597704");insert into Variant(idVariant, Chromosome, Posiion) values("ALG9","17","653246");insert into Variant(idVariant, Chromosome, Posiion) values("AIRE","6","673880");insert into Variant(idVariant, Chromosome, Posiion) values("ALPK3","14","233942");insert into Variant(idVariant, Chromosome, Posiion) values("ALX3","11","154521");insert into Variant(idVariant, Chromosome, Posiion) values("ALDOB","10","113119");insert into Variant(idVariant, Chromosome, Posiion) values("AKAP9","10","364638");insert into Variant(idVariant, Chromosome, Posiion) values("ALOX5AP","7","297914");insert into Variant(idVariant, Chromosome, Posiion) values("AIMP1","16","778588");insert into Variant(idVariant, Chromosome, Posiion) values("ALG2","11","405630");insert into Variant(idVariant, Chromosome, Posiion) values("AGTR1","2","790674");insert into Variant(idVariant, Chromosome, Posiion) values("ALDOA","15","258682");insert into Variant(idVariant, Chromosome, Posiion) values("AIP","6","862987");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH2","6","566896");insert into Variant(idVariant, Chromosome, Posiion) values("AK1","15","301563");insert into Variant(idVariant, Chromosome, Posiion) values("AGXT2","7","449863");insert into Variant(idVariant, Chromosome, Posiion) values("ALX4","15","129256");insert into Variant(idVariant, Chromosome, Posiion) values("ALG6","10","577638");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH7A1","6","697806");insert into Variant(idVariant, Chromosome, Posiion) values("AKR1D1","6","605068");insert into Variant(idVariant, Chromosome, Posiion) values("ALB","18","847289");insert into Variant(idVariant, Chromosome, Posiion) values("ALOX12B","9","684731");insert into Variant(idVariant, Chromosome, Posiion) values("ALG12","9","296319");insert into Variant(idVariant, Chromosome, Posiion) values("AMH","13","308659");insert into Variant(idVariant, Chromosome, Posiion) values("AKT2","Y","333904");insert into Variant(idVariant, Chromosome, Posiion) values("AMELX","2","460519");insert into Variant(idVariant, Chromosome, Posiion) values("ANK1","16","209303");insert into Variant(idVariant, Chromosome, Posiion) values("AMN","11","721342");insert into Variant(idVariant, Chromosome, Posiion) values("ALDH1A3","X","512347");insert into Variant(idVariant, Chromosome, Posiion) values("AGT","9","604189");insert into Variant(idVariant, Chromosome, Posiion) values("AIMP2","7","321857");insert into Variant(idVariant, Chromosome, Posiion) values("ALMS1","3","536716");insert into Variant(idVariant, Chromosome, Posiion) values("ALK","11","222346");insert into Variant(idVariant, Chromosome, Posiion) values("ANGPTL3","11","751265");insert into Variant(idVariant, Chromosome, Posiion) values("AMPD1","13","349233");insert into Variant(idVariant, Chromosome, Posiion) values("AKT3","X","543461");insert into Variant(idVariant, Chromosome, Posiion) values("ALG11","16","780826");insert into Variant(idVariant, Chromosome, Posiion) values("AIPL1","Y","888780");insert into Variant(idVariant, Chromosome, Posiion) values("AMER1","20","399848");insert into Variant(idVariant, Chromosome, Posiion) values("AIFM1","6","173383");insert into Variant(idVariant, Chromosome, Posiion) values("ALX1","6","733706");insert into Variant(idVariant, Chromosome, Posiion) values("ANG","14","841656");insert into Variant(idVariant, Chromosome, Posiion) values("ANGPTL4","16","863830");insert into Variant(idVariant, Chromosome, Posiion) values("ALG3","5","399633");insert into Variant(idVariant, Chromosome, Posiion) values("AKR1C2","17","574716");insert into Variant(idVariant, Chromosome, Posiion) values("AMACR","12","263955");insert into Variant(idVariant, Chromosome, Posiion) values("AGTPBP1","14","778796");insert into Variant(idVariant, Chromosome, Posiion) values("AHR","18","751126");insert into Variant(idVariant, Chromosome, Posiion) values("AMMECR1","6","369275");'

insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCC8","Pulmonary Surfactant Metabolism Dysfunction","0.2049339873168427");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCC11","Dyschromatosis universalis hereditaria 3","0.30240137659449523");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABL1","Pseudohyperkalemia, familial, 2, due to red cell leak","0.14452910690813187");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTN4","Familial hypoalphalipoproteinemia","0.1859132715187647");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTC1","Cone-rod dystrophy 3","0.2715668584326319");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACVR1","Congenital ichthyosis of skin","0.21815952273318967");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCC6","Pulmonary Surfactant Metabolism Dysfunction","0.22472845418676038");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACVR2B","Leukoencephalopathy, progressive, with ovarian failure","0.00832348192147289");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACADSB","Retinitis pigmentosa","0.10392842727140965");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACACA","Alzheimer disease, type 9","0.33674231464904525");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCA3","Gingival fibromatosis with hypertrichosis","0.04336167856044878");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACKR1","Microphthalmia, isolated, with coloboma 7","0.043789909359289114");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD9","Autosomal recessive congenital ichthyosis 4A","0.0037667443987395312");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAT1","Benign recurrent intrahepatic cholestasis 2","0.022780952760113687");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD9","Pulmonary Surfactant Metabolism Dysfunction","0.003044118166047934");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTN1","Progressive familial intrahepatic cholestasis 2","0.16723386517217728");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCA4","Autosomal recessive congenital ichthyosis 4B","0.2770767352181218");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCB11","Inflammatory bowel disease 13","0.2075848072007268");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD8","Pseudoxanthoma elasticum","0.270208269811059");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ADAM9","Progressive familial intrahepatic cholestasis 3","0.17114459152305228");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABO","Alzheimer disease, type 9","0.1471816194944711");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACVR2B","Congenital ichthyosis of skin","0.029070830284284283");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACADSB","Gamma-aminobutyric acid transaminase deficiency","0.2580108241391133");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCB1","Autosomal recessive congenital ichthyosis 4B","0.11855293283139756");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACKR1","Alzheimer disease, type 9","0.28755345648355757");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCD3","Autosomal recessive congenital ichthyosis 4A","0.017144586440120495");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ADAM17","Autosomal recessive congenital ichthyosis 4A","0.2289845775247267");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ADAMTSL2","Charcot-Marie-Tooth disease, type 2","0.3240976592831399");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ADA","Surfactant metabolism dysfunction, pulmonary, 3","0.1596387006183327");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("AARS2","Dyschromatosis universalis hereditaria 3","0.14078901023192947");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACADVL","Tangier disease","0.3059886818362258");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTN2","Congenital ichthyosis of skin","0.10316179021627758");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAT1","Pseudoxanthoma elasticum","0.21782027767540404");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCB7","Autosomal recessive congenital ichthyosis 4B","0.2645803854585376");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACE","Cholestasis, intrahepatic, of pregnancy 3","0.275367492596072");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD9","Hyperlysinemia","0.0941910522141693");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACP5","Combined oxidative phosphorylation deficiency","0.25501283719315637");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACVR2B","Saccharopinuria","0.3216758590854119");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAT1","Inflammatory bowel disease 13","0.12713910596942807");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD9","Pseudohyperkalemia, familial, 2, due to red cell leak","0.059800584191470635");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCA4","Age-related macular degeneration","0.3484841265601574");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACACA","Anemia sideroblastic and spinocerebellar ataxia","0.03537037603846915");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTC1","Retinitis pigmentosa 19","0.2803932296232622");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAD8","Familial hypoalphalipoproteinemia","0.14840879744096916");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCC11","Apocrine gland secretion, variation in","0.09716501124597061");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCD1","Saccharopinuria","0.08718926279658334");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCC2","Autosomal recessive congenital ichthyosis 4B","0.025497036232946683");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAT2","Pseudoxanthoma elasticum","0.24146287590409787");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACAT1","Saccharopinuria","0.004468972548917143");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTG1","Progressive familial intrahepatic cholestasis 3","0.10476398938775176");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACOX2","Charcot-Marie-Tooth disease","0.13922613422802385");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABCG2","Alzheimer\'s disease","0.2165260565934121");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTG2","Pulmonary Surfactant Metabolism Dysfunction","0.3116762533681253");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ABHD5","Alzheimer\'s disease","0.23552489230047513");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ACTN2","Microphthalmia, isolated, with coloboma 7","0.2951708040761929");insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("ADAM17","Microphthalmia, isolated, with coloboma 7","0.12134980819965374");

In [ ]:
insert into Phenotype (Disease_name, Invalid_status) values( "Alzheimer's disease", "I");

In [4]:
pols = "Russian Tatar Ukrainian Bashkir Chuvash Chechen Armenian Other".split()

ss_to_db = []
for i in pols:
    


    #chrom=random.choice(chroms)
    #position = str(int(random.uniform(100100, 900009)))
    #print(position)
    #print(chrom)
    d = random.choice(genes1)
    chrom = random.choice(dis1)
    freq = str(random.uniform(0.001, 0.35))
    
    #insert into Phenotype (Disease_name, Invalid_status) values( "Alzheimer's disease", "I");
    s1 = 'insert into Population(Population_name ) values("'+ i + '");'   
    ss_to_db.append(s1)
    
all_s = ""
for s in ss_to_db:
    all_s=all_s+s

NameError: name 'random' is not defined

In [52]:
all_s

'insert into Population(Population_name ) values("Russian");insert into Population(Population_name ) values("Tatar");insert into Population(Population_name ) values("Ukrainian");insert into Population(Population_name ) values("Bashkir");insert into Population(Population_name ) values("Chuvash");insert into Population(Population_name ) values("Chechen");insert into Population(Population_name ) values("Armenian");insert into Population(Population_name ) values("Other");'

In [47]:
all_s

'insert into Population(VPopulation_name ) values("Russian");insert into Population(VPopulation_name ) values("Tatar");insert into Population(VPopulation_name ) values("Ukrainian");insert into Population(VPopulation_name ) values("Bashkir");insert into Population(VPopulation_name ) values("Chuvash");insert into Population(VPopulation_name ) values("Chechen");insert into Population(VPopulation_name ) values("Armenian");insert into Population(VPopulation_name ) values("Other");insert into Population(VPopulation_name ) values("ethnicities");'

insert into Sample(id, Method, Population_Population_name  ) values(11,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(12,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(13,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(14,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(15,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(16,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(17,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(18,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(19,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(20,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(21,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(22,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(23,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(24,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(25,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(26,"Panel","Russian");

insert into Sample(id, Method, Population_Population_name  ) values(84,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(85,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(86,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(87,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(88,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(89,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(90,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(91,"Panel","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(92,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(93,"Exome","Tatar");


insert into Sample(id, Method, Population_Population_name  ) values(124,"Exome","Armenian");insert into Sample(id, Method, Population_Population_name  ) values(125,"Panel","Ukrainian");insert into Sample(id, Method, Population_Population_name  ) values(126,"Panel","Other");insert into Sample(id, Method, Population_Population_name  ) values(127,"Panel","Russian");insert into Sample(id, Method, Population_Population_name  ) values(128,"Exome","Other");insert into Sample(id, Method, Population_Population_name  ) values(129,"Exome","Chechen");insert into Sample(id, Method, Population_Population_name  ) values(130,"Exome","Other");insert into Sample(id, Method, Population_Population_name  ) values(131,"Exome","Bashkir");insert into Sample(id, Method, Population_Population_name  ) values(132,"Exome","Russian");insert into Sample(id, Method, Population_Population_name  ) values(133,"Exome","Chechen");insert into Sample(id, Method, Population_Population_name  ) values(134,"Exome","Other");insert into Sample(id, Method, Population_Population_name  ) values(135,"Exome","Armenian");insert into Sample(id, Method, Population_Population_name  ) values(136,"Exome","Bashkir");insert into Sample(id, Method, Population_Population_name  ) values(137,"Exome","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(138,"Panel","Other");insert into Sample(id, Method, Population_Population_name  ) values(139,"Panel","Tatar");insert into Sample(id, Method, Population_Population_name  ) values(140,"Exome","Bashkir");insert into Sample(id, Method, Population_Population_name  ) values(141,"Exome","Chuvash");insert into Sample(id, Method, Population_Population_name  ) values(142,"Panel","Ukrainian");insert into Sample(id, Method, Population_Population_name  ) values(143,"Exome","Chechen");

insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(11,78,"ACP5","AACAGATTA/CTCCGCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(11,71,"ACTN4","CAG/AATCACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(11,93,"ACOX2","AGAATCC/GGTGTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(11,73,"ABCA7","TGTGTG/GGTAAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,52,"ADAMTS18","T/TT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,71,"ABAT","TTA/CCCACGCCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,60,"ABCA12","GCTTAGTA/TTTATAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,67,"ABCC8","GTGAAATTA/CG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,85,"ACTB","-/TGGTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,28,"ACVR1","AACCGTC/AA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,66,"ADAM9","TCCGT/TTCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(12,41,"ACTG2","CTCCCGCT/ATAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(13,100,"ABCC8","-/TGCGGTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(13,40,"ABCD4","TCTC/TG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(13,129,"ACVRL1","CCAATA/TGCGTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,13,"ACADSB","GTC/TGCCCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,69,"ACTG1","ATATAGGGC/GCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,75,"ACP5","GTATAAAAC/TG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,95,"ABCB7","G/CCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,3,"ABO","TAGCGAA/AGTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,146,"ADAMTS13","CAGGCGTTT/TGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,8,"ACTL6B","A/AACCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,34,"ABCD1","ACCCCGA/ATGCTAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,108,"ADA","G/ACGATCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,53,"ACVRL1","-/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(14,14,"ABCB4","ACC/CG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,140,"ABCD1","ATGG/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,43,"ADAMTS18","-/CATCACC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,30,"ACY1","G/TCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,89,"ABCG8","A/GTGGCACCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,137,"ABCB7","GCTCA/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,15,"ACHE","AGCGG/TGTTTTTAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,9,"ACOX1","GTATAA/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,83,"ACTN3","GGTCGC/AGCCTTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,110,"ABCG2","GT/GCGAAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,114,"ABO","TGTTCG/TCCCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,113,"ABCA7","TCTC/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,110,"ABCG5","-/TAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,66,"ABCD3","CTGCGCTA/AG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(15,3,"ABCC6","TGCAAT/TCCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,105,"ACOX1","TTAC/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,105,"ACTC1","GGTGTGGGG/ATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,148,"ABCD3","TAATTAC/ACGGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,32,"ABO","GC/TA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,78,"ABCA4","TC/AC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,141,"ABCC2","CGATTCCCA/AAGTTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,102,"ABCC8","TAAGG/ATGCTCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,86,"ACHE","CCGCGAG/TG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(16,117,"ABCB1","AA/CCTTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,98,"ACADVL","CCCTCTT/AAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,122,"ACADS","CC/GTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,103,"ACO2","AT/TG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,31,"ABCB4","CTTCCGTTT/CGCCTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,38,"ACTA2","T/GTCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,116,"ABHD5","ATCCA/TTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,9,"ACAD9","GTACT/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,109,"ACVRL1","AAA/CA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,85,"ACTA1","TTGATCGT/GAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,65,"ABCB1","GCCAG/ACCTCAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,17,"ABCD1","AA/CA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,49,"ABCB7","-/AGATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,64,"ADAM9","GGAAGTT/GGATCTCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,112,"ACVR1","CTCGATGTG/GGTCTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,54,"AASS","ACTC/AGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(17,140,"ACADM","-/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(18,28,"ADAMTS2","GTGGGACA/CGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(18,133,"ABCD1","-/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(18,101,"ACTA2","ACATC/GTCTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(18,115,"ABCA5","TGAA/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,132,"ACHE","-/CACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,72,"AASS","TTCGTAGA/TGGAGTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,54,"ACVRL1","G/CCGGAGGGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,82,"ADA2","C/TCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,110,"ACER3","GTGA/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,6,"ABCB7","G/TTGCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,141,"ABCG8","AA/CAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,85,"ACAN","CTAATCCC/GAATC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,113,"ABCD3","C/CAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,68,"ACVR2B","GTC/GATCCAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,29,"AARS2","AACCAC/ATAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,129,"ACD","A/CTCCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,123,"ACADVL","ACGCCATC/TTTGAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,14,"ABCA12","GTGGT/CGCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(19,89,"ABCG2","TTACACCG/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,35,"ABCG8","AC/TTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,54,"ADAMTS3","ATGGTCTCG/GCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,147,"ACADSB","AAG/AGCAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,7,"ABCB1","AG/CCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,49,"ADAMTSL2","CA/TATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(20,21,"ACER3","A/TC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,124,"ABO","AGA/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,51,"ACOX1","ATA/CCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,43,"ADA","CTGGCC/CCGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,30,"ACER3","ATATGCA/GGTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,36,"ABCD3","C/TCCACGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,34,"ABCB6","GTGGGC/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,20,"ADAM9","TTG/TAGCTAACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,95,"AASS","CCTTATC/GTCGAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(21,107,"ACSL4","GT/AGTAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,73,"ABCD1","ACCTTA/CCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,142,"ADAMTS13","AT/CTGAGCGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,41,"ACAD9","AGCG/TCTACACCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,39,"ABO","-/AGCCACTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,66,"ABHD5","G/GTCTAAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,141,"ACP2","ACGCAT/TAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,78,"ACTG2","-/TAAGAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,96,"ADAM22","GGCGGA/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,74,"ADAMTS10","CGGAT/CTCATGTTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(22,56,"ACP5","G/CTGTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(23,39,"ACTG1","CGCGCCGA/TTGCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(23,64,"ACVR2B","-/ACACATGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(23,27,"ABCC11","ACG/ATGCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(23,132,"ADAM10","A/ATCGGACGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(23,19,"ADAMTS10","TCTGATC/GCTTTCTGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,68,"ADA2","AGAA/TAGGAGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,80,"ACAD9","-/GAATCAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,105,"ACADS","TTG/TCCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,148,"ADAM22","TAGACGTAT/GAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,107,"ACAT2","TTATCTGGG/AGCCTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,4,"ABAT","CTTCT/CACTCCCCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(24,142,"ACOX2","GACGGT/GTCGAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,149,"ABHD5","TTGAA/ATTACGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,40,"ACO2","TGT/ACCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,99,"ACVR1","A/TTTTAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,67,"ACD","-/CTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,92,"ADAMTS13","CGA/GA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,72,"ABCG2","-/CGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,106,"ADAM10","CTGTCCGCA/GT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,111,"ACTG1","AC/TACTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,133,"ABCA3","A/ATTTGGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,138,"ACAD9","ATGGGCGG/ACCGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,125,"ABCB7","GAATC/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,100,"ADAMTS10","GCGCCAACG/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,9,"ACAT2","A/AT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,88,"ACP2","-/GTCAGCCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(25,4,"ACTN2","TGTAT/AAGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,23,"ADAM17","CT/CATTGACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,102,"ACAN","GATGGGCCC/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,17,"ACAD9","TGACTTCG/TCCTGAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,83,"ADAMTS18","CTGTC/TGCGTTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,57,"ABCA1","-/ATGCTCGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,32,"ACVR1","ATCACG/AA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,135,"ABCA12","-/GGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,75,"ACTG2","-/ATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,125,"ABO","GTT/TT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,96,"ACTA2","GA/ATCATGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,7,"ADAM10","T/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,23,"ACOX1","TGTTGAGTC/AGGTGAGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,30,"ABCA5","ACTTAAAT/GTATGGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(26,48,"ACVR2B","TTGGTCTA/ATT");

 insert into Population(Population_name ) values("Russian");insert into Population(Population_name ) values("Tatar");insert into Population(Population_name ) values("Ukrainian");insert into Population(Population_name ) values("Bashkir");insert into Population(Population_name ) values("Chuvash");insert into Population(Population_name ) values("Chechen");insert into Population(Population_name ) values("Armenian");insert into Population(Population_name ) values("Other");

In [44]:

lines = open('gene_condition_source_id').readlines()
print(lines[90].split("\t"))


for i in range(10, 150):
    

SyntaxError: unexpected EOF while parsing (<ipython-input-44-4dec7f4cf452>, line 6)

insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(84,52,"ABCC2","ATTCATA/AA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(84,67,"ABCA4","GACACTGCG/CG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,83,"ACTB","AGCGGT/TGACTGTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,65,"AARS2","-/TCCAGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,143,"ABAT","-/CCAACAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,122,"ACADVL","TCGAAGG/TCTGAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,55,"ABCA1","T/AGGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,4,"ADA2","-/ACATGATGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,58,"ACTN4","-/GCTCTAAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,58,"ACP2","CACTTGA/ATAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,93,"ABL1","C/TT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,66,"ABCA5","GC/ATGCTACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,134,"ACTN2","CGTG/GGCCCAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,12,"ACTN3","G/TTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,38,"ABCB6","GGGCT/AAAGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,42,"ABCC9","AC/ACTTCGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,116,"ABCG2","CGCCAA/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(85,130,"ABCC8","TCA/ATA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,34,"ACVR1","GACC/GCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,37,"ACTA1","-/TCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,54,"ACTG1","GCAA/GTTTCCCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,30,"ACSF3","TGGAGGCCC/AAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,37,"ACTL6B","GTTCAAA/GCCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,98,"ABCA4","ATTT/TAACC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,124,"ACAD9","A/TTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(86,140,"AARS2","TACATG/TAAGTATA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,141,"ABCB6","CGCCCGG/CCGCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,127,"ACADS","CC/CAGGCTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,134,"ADAM9","GCCTCGGT/TTCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,96,"ABHD12","TGAAGTC/CACGAAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,126,"ACAD8","GCATAC/GCCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,48,"ABCB7","GTGTG/GAACGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,109,"ACP2","GTTTTTAG/GCGTATA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,74,"ACAN","ACGCTTC/AGTCTTGAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,119,"ACADVL","TATAA/CGTCGCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,65,"ABCG2","TG/TCTGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(87,85,"ACVR2B","AGGG/AGCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,90,"ABCA7","AGCTATAT/TG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,49,"ABCC11","CGG/CCAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,19,"ACHE","CGTCCGC/ATG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,63,"AARS2","ATGCT/GAGTACT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,38,"ACTA1","-/CCAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,136,"ABCC9","-/TGCAGCAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,71,"ACADVL","GCCAG/CAAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,144,"ABCG8","AGGAGAA/TTAGAGCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,90,"ACSL4","C/TCGGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,141,"ABCC8","CATTAATGA/ACCGTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,44,"ABCB6","TA/AGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,4,"ADAM22","GAGTGT/CG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(88,10,"ABO","-/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(89,83,"ACD","-/GAGCAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(89,120,"ABAT","CG/TAAGAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(89,97,"ACOX2","-/GTGTAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(89,139,"ABHD12","TA/TACCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,143,"ACTN4","CATCC/CCGCGCTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,3,"ADAMTSL2","C/TAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,34,"ACP4","GCA/GC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,104,"ADAMTS10","GTGTGTT/TC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,26,"ABCC11","CT/CGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,10,"ACHE","AAGTA/CTAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,19,"ACE","AATA/CAGGCTTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,73,"ACOX2","CGCG/CTGCAGCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,79,"ACO2","GC/ACTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(90,101,"ADAMTS17","-/TCCTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,59,"ABL1","TTCT/AGGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,33,"ACER3","-/ACGGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,139,"ACACA","TGACCTAAT/GCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,6,"ADAM22","AGG/AG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,93,"ACY1","GGAAATG/GGTTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,139,"AARS2","T/CATAACAAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,112,"ADAM17","GATTAC/CGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,80,"ACTA2","CAC/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,44,"ABCB7","TGTATG/TGGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,107,"ABCC9","GAGTGGC/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(91,149,"ACAN","GG/AAAACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,130,"ABCA12","GGACCC/AGAGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,71,"ACADM","CACG/ATATGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,65,"ACSL4","CC/ACATA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,133,"ABCG2","TAACTA/GATATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,132,"ADAM9","TGACC/CTGGGTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,77,"ABCA7","-/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,64,"ADAMTSL2","ATTAC/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,39,"ABCA4","CAC/AC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,111,"ACTL6B","CGAAGTCGT/GAAGCAGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,144,"ABCC2","CAGTGCCA/GTCTACCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,123,"ADAM10","TC/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,129,"ABCG8","TCGGGTCTA/AT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,81,"ACADVL","C/TGTCGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,98,"ABCB4","TCCAA/AAAGTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,112,"ADAMTS18","CTCTT/CATCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(92,56,"ACTB","TAA/TGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,41,"ACTN3","ACGG/GGCATATAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,121,"ACTA2","TT/TGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,55,"ADAMTS13","ATCTTTCTT/GTCAGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,131,"ACSL4","GAAAGGCC/CCCTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,78,"ACHE","AG/TTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,44,"ADAM9","CCCACA/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,51,"ACOX1","CTTTG/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,133,"ACSF3","CAACCAG/CATTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,82,"ACAT2","ATGAACGCG/ACAGGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(93,140,"ACADM","TCCGC/AGC");


insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,5,"ACP2","TGGGA/TTCTCAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,31,"ABHD12","AG/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,145,"ABCA7","CAGGT/GT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,79,"ACSL4","TTG/GAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,70,"ACP5","TA/AAAACCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,78,"ACTB","T/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,25,"ACTN2","T/TGATACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,42,"ADAMTS17","CGT/GTCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(124,114,"ACTL6B","ACTC/TCCAGGGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,117,"ABCA1","T/CCTCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,16,"ACVR1","TCAACGCT/TC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,103,"AASS","GGTAGAAAT/TGAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,53,"ACHE","CGGC/TAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,87,"ADAMTS3","TTCTTCC/TTGCCAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,82,"ACSF3","-/ATTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,56,"ACY1","-/AGTAGACT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,23,"ABCC9","TTCGCGG/TTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,84,"ACTB","TGCC/GAGGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(125,38,"ABCB11","TTGCAGTG/AACAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,52,"ACO2","TTGGCTA/CAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,34,"ACADSB","TTT/CGGCGGTTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,54,"ADAMTS3","GCTTAGAT/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,60,"ACY1","C/CTAGAGGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,149,"ACTN1","GCATATC/GC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,148,"ADAM17","GAGCC/AA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,64,"ACKR1","A/AAGCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,139,"ACP2","ATGC/ACGAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,44,"ABCA1","G/TATCTCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,25,"ACP5","-/TGGCGCCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,41,"ACP4","TTC/CTTGGTAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,39,"ACTG2","GCGTAAAGA/TAACCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,98,"AARS2","CTAAACG/TCATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(126,37,"ABCG2","C/GCGTTCGGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,118,"ADAMTS10","T/CCTTACC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,5,"ACADVL","CTT/GAGGGGGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,104,"ACP5","GA/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,89,"ACADSB","A/AGCAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,81,"ABCB7","CGC/CCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,25,"ADAM9","GGGCT/ATACTTTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,71,"ABCD3","GATT/GACTGTTCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,57,"ABCB11","GA/TCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,88,"ACER3","ATGCCTAGA/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,13,"ACTC1","GTTACAA/ATGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,91,"ADA2","ATTTA/CCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,8,"ABCG8","ACACTG/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,87,"ACAT1","AT/GCGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(127,75,"ACVR2B","-/GCTGTCTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,139,"ABCC9","AACTCA/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,38,"ABHD12","C/AGAGCTGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,80,"ABCB6","AAC/GG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,70,"ACTB","T/CCATCTACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,97,"AARS2","ACAAAT/AGGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,51,"ABCA5","GCACC/GGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,3,"ACAN","-/CTAGATA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(128,149,"ADAMTS18","TC/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,34,"ACAT2","GAGATTCA/CGAAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,39,"ABCA5","CTTC/ACCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,49,"ADAMTS17","CGTCTATG/TGTTCTCTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,144,"ABCD1","GTGGA/AGCCACAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,29,"ABCC6","A/TTATCGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(129,113,"ABCG5","AATCT/TTGCTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,48,"ACADSB","ACT/AAAACCTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,7,"ADAMTSL2","GGGAC/CCTCTTCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,142,"ACTN2","GTGAC/CGCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,49,"ABCA7","AAT/AGAAGACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,83,"ACTB","CGCCT/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,15,"ACTG1","CATTTGA/TTCCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,54,"ABCA5","TGATTG/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,46,"ABHD5","AAGATGTT/TCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,4,"ABCC6","AGTCGGCC/GGGTCTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,138,"ABCG8","G/ACACTTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,16,"ABL1","ATAGTTGTT/ACACACATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(130,58,"ABCB11","CGTTTACCT/ATACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(131,36,"ACADS","CGC/CGTAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(131,7,"AARS2","GGGAGCGAC/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(131,32,"ABHD5","CGCTTC/ACGCTGCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(131,26,"ABCA4","GTTAGTCAA/CCCTGTCCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(132,75,"ADAMTS10","AGG/AA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(132,81,"AARS2","TT/GACTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,123,"ABO","AATG/ATCAGGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,146,"ABCB1","CC/CTAGCGGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,81,"ACAD9","A/TGGCAGGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,138,"ACTN4","GTGCCC/ATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,8,"ABHD5","AAA/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,116,"ACTB","TGCGCA/GCATCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,30,"ACTA1","TG/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,97,"ABCB7","C/ATCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,112,"ACOX2","CATCT/GAGAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,45,"ABCC6","CAGTCCCC/CCAACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,134,"ACADVL","A/GTCAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(133,102,"ADAMTS10","C/CAATGTGCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,67,"ACTB","A/ATTGGAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,133,"ABCD4","AGTA/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,100,"ADAM9","ACGC/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,19,"ACADM","-/GGGTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,132,"ACY1","G/ATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,31,"ACP2","CCCCTACAC/GACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,3,"ADAM10","CA/AGGCGCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,88,"ADAM17","CTTTAGC/TGTTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,114,"ACO2","TA/CGAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,132,"ABCA7","CTCC/GGGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,131,"ACADSB","TG/ATGCAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,107,"ADA2","GAGTGTGCG/GTTAAATTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,16,"ACVRL1","A/CTCAGAAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,133,"ABCA12","T/AG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,145,"ACSF3","GAC/ACACGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(134,103,"ACTL6B","GAGGAGG/TA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,9,"ACSF3","GTTCAT/AGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,54,"ACVRL1","CGCTGTA/ACA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,41,"ABCB11","GCGATTCCC/TAACCTCCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,40,"ACKR1","TGG/GGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,88,"ABCC8","CGTGG/AGCGTCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,15,"ACADM","-/GGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,123,"ACER3","CGTAGG/CCCTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,125,"ACOX1","CGAA/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,137,"ADAM9","CC/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,57,"ADA2","TTG/TAAGAAAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,82,"ADAMTS13","CCGCAC/ACTTCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,91,"ABCG2","AGA/TTTGTCACC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(135,146,"ADAMTS2","AAAAAT/AGAGTCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(136,49,"ABCB4","TGGAGT/CAGCAACT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(136,4,"ACTA2","AGCCA/TCGCTACG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(136,139,"ABCB7","CAGGAC/ATG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(137,15,"ACSF3","CAAAATTTG/ATATCACTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(137,17,"ABCA7","CGATAC/TCGGGGCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(137,138,"ACTC1","GTCCT/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(138,28,"ADAMTS2","GTTCTGTTG/GCTTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(138,10,"ABCG2","GC/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(138,103,"ACTN1","GACTATAG/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,87,"ABL1","GCGACCAAA/TCCGGAAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,118,"ACKR1","GCCCG/TTAATC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,51,"ADAM9","AACCT/GTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,110,"ACSL4","GTC/AGG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,64,"ACSF3","TTTTGCT/AAACGAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,62,"ACTN1","TT/TGTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,129,"ACADS","AAATTTTA/TCATATG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,10,"ABCG2","-/T");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(139,24,"ADAMTS18","AGTG/TGGTCGCGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,59,"ACO2","CAAGTA/CGGTAAAT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,113,"ABCB11","CA/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,147,"ABCC11","GTGCATC/GAGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,137,"ABCA1","G/TACGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,47,"ADAM22","AGAT/AGTTCGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,44,"ACER3","CT/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,49,"ABL1","ATTCGC/AAGAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,64,"ACAD8","-/AGGGTA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,91,"ACVR2B","CT/TC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,105,"ABHD12","-/TTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,75,"ABCB7","CCTAAAC/TGCCATGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,12,"ACTN2","TATAATCGT/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,15,"ACAT2","CACG/GTCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,145,"ABCA4","CG/CC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,14,"ADA2","CCGTTGGG/TTATCC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,129,"ACAT1","AAGTCGAA/GCCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,68,"ABCD1","ACAGTA/C");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,79,"ABCA7","-/GCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(140,119,"ACADSB","GAAAG/GGGTTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,98,"ADA","TTACCGTCA/ATCCACT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,85,"ACD","GGCA/GT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,93,"ACOX1","CACCT/AACAC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,66,"ACACA","CTAAGATG/CTGTTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,11,"ACTB","AAGTTCC/TACACGTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,14,"ACADM","TGTAT/CGTACCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,103,"AASS","TGTTCTTA/GTATCTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,98,"ACP5","-/CTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,18,"ADAM9","TCG/CTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,22,"ACAT1","GGACGACTG/GTTATCA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,50,"ABCG2","TTTAGGGC/CTCGGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,41,"ACP4","CAAGAGC/TCTGGGAGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,129,"ABAT","ACTTC/GTGTTTT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(141,102,"ADA2","ATAGGAGG/GCAGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,51,"ABCD4","T/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,142,"ACSF3","A/GCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,54,"ACTA1","-/ATAACTAG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,125,"ACE","CCACGG/TTGCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,96,"ADAMTS17","ACCTGG/GAGCAGTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,16,"ACADM","T/GC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,52,"ABAT","CG/ATAGCATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(142,42,"ABCC9","TGCTACC/AAACGAGGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,133,"ADA2","GATTAAGC/AAGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,88,"AASS","AGAAAT/ACATATCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,17,"ABAT","AGAGATA/GTG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,14,"ACO2","CGCCTCC/CATATT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,118,"ADAM10","ACGGGACA/GGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,125,"ACP5","TGGATTAT/TCG");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,24,"ABCC8","GGCGCCG/AACTAAAA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,5,"ABCA7","CTGCGCTA/A");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,114,"ACOX1","ACC/GCCAACCT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,37,"ADAM17","-/G");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,78,"ACVRL1","AGGAAGT/GTC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,28,"ABCB11","ACT/AT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,51,"ADAMTS17","AGAGG/AATGA");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,64,"ABHD12","TCGCTAAAA/CTGT");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,90,"ACTA2","G/GCGTGC");insert into Sample_variant(Sample_id,DP, Variant_idVariant,GENOTYPE ) values(143,84,"ACAD9","TACCC/TCTTT");

In [ ]:
all_s = ""
ss_to_db = []
start_ind = 11
for i in range(16):
    


    #chrom=random.choice(chroms)
    #position = str(int(random.uniform(100100, 900009)))
    #print(position)
    #print(chrom)
    d = random.choice(genes1)
    chrom = random.choice(pols)
    freq = str(random.uniform(0.001, 0.35))
    
    #insert into Phenotype (Disease_name, Invalid_status) values( "Alzheimer's disease", "I");
    s1 = 'insert into Phenotype_Variant(Variant_idVariant, Phenotype_Disease_name, Frequency ) values("'+ d+ '","' +chrom +'","'+freq + '");'   
    ss_to_db.append(s1)
    start_ind = start_ind+1